# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xfsprogs/hotfix-off_t.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- xfsprogs-6.13.0/mkfs/proto.c.vanilla	2025-03-20 10:17:29.920584672 +0100
+++ xfsprogs-6.13.0/mkfs/proto.c	2025-03-20 10:17:52.521512583 +0100
@@ -20,7 +20,7 @@
 static void rsvfile(xfs_mount_t *mp, xfs_inode_t *ip, long long len);
 static int newregfile(char **pp, char **fname);
 static void rtinit(xfs_mount_t *mp);
-static long filesize(int fd);
+static off_t filesize(int fd);
 static int slashes_are_spaces;
 
 /*
